### PR TITLE
[zh] Sync a task: egress/wildcard-egress-hosts/

### PR DESCRIPTION
**Pull request by windsonsea**
Sync with the en text:

```
content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md
```

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md [hunk 1]
```diff
@@ -59,7 +80,7 @@ Egress 流量，如：`edition.cnn.com`。本示例描述如何为通用域中
     $ export SOURCE_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name})
     {{< /text >}}
 
-## 引导流量流向 Wildcard 主机   {#configure-direct-traffic-to-a-wildcard-host}
+## 配置流向 Wildcard 主机的引导流量   {#configure-direct-traffic-to-a-wildcard-host}
```
Comment (my-git9) [inline content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md:83]:
```suggestion
## 配置流向 Wildcard 主机的引导流量   {#configure-direct-traffic-to-a-wildcard-host}
```
“引导”从动词改成了名词，感觉原意变了

Comment (windsonsea) [reply content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md]: Thanks, reverted.

File: content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md [hunk 2]
```diff
@@ -20,20 +20,41 @@ Egress 流量，如：`edition.cnn.com`。本示例描述如何为通用域中
...
+
+{{< tab name="Istio API" category-value="istio-apis" >}}
+
+{{< text bash >}}
+$ istioctl install --set profile=demo --set meshConfig.outboundTrafficPolicy.mode=REGISTRY_ONLY
+{{< /text >}}
+
+{{< tip >}}
+您可以在 `demo` 配置文件以外的 Istio 配置上运行此任务，
+只要您确保 [部署 Istio Egress 网关](/zh/docs/tasks/traffic-management/egress/egress-gateway/#deploy-Istio-egress-gateway)。
```
Comment (my-git9) [inline content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md:39]:
```suggestion
只要您确保[部署 Istio Egress 网关](/zh/docs/tasks/traffic-management/egress/egress-gateway/#deploy-Istio-egress-gateway)。
```

File: content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md [hunk 3]
```diff
@@ -100,88 +121,165 @@ Egress 流量，如：`edition.cnn.com`。本示例描述如何为通用域中
     <title>Wikipedia – Die freie Enzyklopädie</title>
     {{< /text >}}
 
-### 清除引导流量至 wildcard 主机  {#cleanup-direct-traffic-to-a-wildcard-host}
+### 清理到 Wildcard 主机的引导流量  {#cleanup-direct-traffic-to-a-wildcard-host}
```
Comment (my-git9) [inline content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md:124]:
```suggestion
### 清理将流量引导至 Wildcard 主机的规则  {#cleanup-direct-traffic-to-a-wildcard-host}
```

File: content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md [hunk 4]
```diff
@@ -100,88 +121,165 @@ Egress 流量，如：`edition.cnn.com`。本示例描述如何为通用域中
...
 
 {{< text bash >}}
 $ kubectl delete serviceentry wikipedia
 {{< /text >}}
 
-### 单一 hosting 服务器的 Wildcard 配置  {#wildcard-configuration-for-a-single-hosting-server}
+## 配置到 Wildcard 主机的 Egress 网关流量 {#configure-egress-gateway-traffic-to-a-wildcard-host}
```
Comment (my-git9) [inline content/zh/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md:130]:
```suggestion
## 配置到 Wildcard 主机的 Egress 网关流量规则 {#configure-egress-gateway-traffic-to-a-wildcard-host}
```

**Review by my-git9 — APPROVED** [review]
LGTM

Thanks

**Pull request merged**